### PR TITLE
Added default ezplatform.yml file from eZStudioDemoBundle

### PR DESCRIPTION
**Pull request by kamilmusial**
ping @lserwatka

### CODE REVIEW

File: app/config/ezplatform.yml [hunk 1]
```diff
@@ -23,4 +27,23 @@ ezpublish:
...
             # need to manually modify your database data to reflect this to avoid exceptions.
             var_dir: var/site
-            languages: [eng-GB]
+            languages:
+                - eng-GB
+                - fre-FR
+                - ger-DE
+                - nor-NO
+        fr:
+            var_dir: var/site
```
Comment (andrerom) [inline app/config/ezplatform.yml]: not needed, duplicates site_group setting

Comment (lserwatka) [inline app/config/ezplatform.yml]: Yes, updated version is coming.

**Comment by kamilmusial** [comment]
@andrerom @lserwatka  Please take a look

**Comment by lserwatka** [comment]
+1

**Pull request merged**